app/uk/gov/hmrc/brm/models/response/Record.scala

```diff
@@ -25,14 +25,14 @@ import uk.gov.hmrc.brm.utils.BRMLogger
 
 case class Record(child: Child, status: Option[StatusInterface] = None) {
 
-  private def processFlags = BrmConfig.processFlags
+  private def processFlags = true
```

Comment:
This has gone from config to a fixed value, I know defaults are true but not sure we want this fixed?

Reply:
This was always true and never changed in any of the environment config. Using config inside of a case class is poor so it'll require a proper refactor if you want to re-introduce the (unused) flagging code